### PR TITLE
cthub-247--frontend-validation

### PR DESCRIPTION
**Pull request by tim738745**

### CODE REVIEW

**Review by naomiaro — CHANGES_REQUESTED** [review]
Are you using Vscode @tim738745 ? If you install the `prettier` extension the code formatting should format on save.

File: frontend/src/components/Form.js [hunk 1]
```diff
@@ -274,19 +388,34 @@ const Form = () => {
               <TextField
                 id="drivers_licence"
                 inputProps={{ maxLength: 9 }}
```
[the remark this answers is not in the document]
Comment (tim738745) [reply frontend/src/components/Form.js:390]: Sure; and yes I am using vscode; will install prettier.

**Comment by naomiaro** [comment]
@tim738745 you'll have to rebase this branch off the upstream release 1.0.0 so that it can be merged.

**Comment by tim738745** [comment]
@naomiaro I think I rebased correctly

**Comment by naomiaro** [comment]
hmm this should only have commits from yourself and not Kuan etc. Usually it will require a force push to the PR as the history will be rewritten.

**Comment by tim738745** [comment]
ohh after I rebased and tried to push vscode wanted me to pull stuff into my branch which I did instead of force pushing...let me see if I can fix this

**Comment by naomiaro** [comment]
Yeah the vscode UI might pull by merging, I don't usually use vscode so I'm not sure. In the terminal you can use `git pull --rebase` to pull changes into a branch.

**Comment by tim738745** [comment]
ok it should be good now

**Review by naomiaro — APPROVED** [review]

**Pull request merged**